scrapy/spiders/crawl.py

```diff
@@ -55,6 +57,8 @@ def _build_request(self, rule, link):
 
     def _requests_to_follow(self, response):
         if not isinstance(response, HtmlResponse):
+            logger.debug("Skipping non-html response %(response)s" , 
```

Comment:
I think it should use self.logger

Reply:
Ok. I'll try to fix it as soon as I have time.